src/balatro_gym/interfaces.py

```diff
@@ -73,22 +73,22 @@ class PokerScale:
 
 
 class PokerHandType(Enum):
-    HIGH_CARD = PokerScale(1, 10)
-    PAIR = PokerScale(1, 15)
+    HIGH_CARD = PokerScale(1, 5)
+    PAIR = PokerScale(2, 10)
     TWO_PAIR = PokerScale(2, 20)
     THREE_SET = PokerScale(3, 30)
     FULL_HOUSE = PokerScale(4, 40)
-    FLUSH_HOUSE = PokerScale(3, 50)
-    FOUR_SET = PokerScale(3, 30)
-    FIVE_SET = PokerScale(3, 35)
-    FLUSH = PokerScale(2, 15)
+    FLUSH_HOUSE = PokerScale(14, 140)
```

Comment:
Gosh, I don't know what I was referencing when I wrote this, but it clearly wasn't the official Balatro poke hands reference!